signer/src/testing/wsts.rs

```diff
@@ -30,6 +30,22 @@ pub struct SignerInfo {
     pub signer_public_keys: BTreeSet<PublicKey>,
 }
 
+/// Generate a set of public keys for a group of signers
+pub fn generate_signer_set<R>(rng: &mut R, num_signers: usize) -> Vec<PublicKey>
```

Comment:
[Suggestion] I might call this something more like `generate_signer_set_public_keys` since you're only returning the public keys, or even something like this to be able to get it from any `Vec<SignerInfo>`.
```rust
pub trait SignerSetInfoExt {
    fn to_public_keys(&self) -> Vec<PublicKey> {
        ...
    }
}
impl SignerSetInfoExt for Vec<SignerInfo> {}
```

Reply:
Oops forgot to update. Hmmm, think I'll update this function name in https://github.com/stacks-network/sbtc/pull/555 if you don't mind.